pages/index.js

```diff
@@ -1,108 +1,33 @@
-import ExercisesList from "@/components/ExercisesList";
-
-import FilterList from "@/components/FilterList";
-import { useState } from "react";
+import HomeHeading from "@/components/HomeHeading";
+import HomeLink from "@/components/HomeLink";
+import HomeText from "@/components/HomeText";
 import styled from "styled-components";
 
-export default function HomePage({ exercises, muscleGroups }) {
-  const [filterMode, setFilterMode] = useState(false);
-  const [selectedMuscleGroups, setSelectedMuscleGroups] = useState([]);
-  const [filteredExercises, setFilteredExercises] = useState(exercises);
-  const [muscles, setMuscles] = useState(muscleGroups);
-
-  function handleShowFilter() {
-    setFilterMode(!filterMode);
-  }
-
-  function handleSelect(muscleGroup) {
-    if (!selectedMuscleGroups.includes(muscleGroup)) {
-      const newSelectedMuscleGroups = [...selectedMuscleGroups, muscleGroup];
-      setSelectedMuscleGroups(newSelectedMuscleGroups);
-
-      setFilteredExercises(
-        filteredExercises.filter((filteredExercise) =>
-          newSelectedMuscleGroups.every((selectedMuscleGroup) =>
-            filteredExercise.muscleGroups.includes(selectedMuscleGroup)
-          )
-        )
-      );
-
-      setMuscles(muscles.filter((muscle) => muscle !== muscleGroup));
-    }
-  }
-
-  function handleDeselect(muscleGroup) {
-    const newSelectedMuscleGroups = selectedMuscleGroups.filter(
-      (selectedMuscleGroup) => selectedMuscleGroup !== muscleGroup
-    );
-    setSelectedMuscleGroups(newSelectedMuscleGroups);
-
-    setFilteredExercises(
-      exercises.filter((exercise) =>
-        newSelectedMuscleGroups.every((selectedMuscleGroup) =>
-          exercise.muscleGroups.includes(selectedMuscleGroup)
-        )
-      )
-    );
-    const newMuscles = [...muscles, muscleGroup];
-    newMuscles.sort((a, b) => a.localeCompare(b));
-    setMuscles(newMuscles);
-  }
-
-  function handleClear() {
-    setSelectedMuscleGroups([]);
-    setFilteredExercises(exercises);
-    setMuscles(muscleGroups);
-  }
-
-  return (
-
-    <StyledSection>
-      <H1 >WELCOME TO YOUR EXERCISE LIST</H1>
-      <FilterButton type="button" onClick={handleShowFilter}>
-        Filter ☰
-      </FilterButton>
-      {filterMode ? (
-        <FilterList
-          muscleGroups={muscles}
-          onSelect={handleSelect}
-          selectedMuscleGroups={selectedMuscleGroups}
-          onDeselect={handleDeselect}
-          onClear={handleClear}
-        />
-      ) : null}
-      <ExercisesList exercises={filteredExercises} />
-    </StyledSection>
-  );
-}
+const HomeBody = styled.section`
+  background-color: #fac74e;
+  min-width: 100%;
+  min-height: 100%;
+  position: absolute;
+  box-sizing: border-box;
```

Comment:
Is there a need to redefine it? You have box-sizing: border-box already defined in you styles.js